packages/synapse-react-client/src/utils/context/FullContextProvider.tsx

```diff
@@ -41,7 +41,7 @@ export function FullContextProvider(props: FullContextProviderProps) {
     <QueryClientProvider client={queryClient ?? defaultQueryClient}>
       <ThemeProvider theme={theme}>
         <SynapseContextProvider synapseContext={synapseContext}>
-          {children}
+          <Suspense fallback={null}>{children}</Suspense>
```

Comment:
Added a Suspense to the wrapper object used by SWC, just in case an unwrapped component ends up being used in SWC.

Reply:
Random thought: it would be so nice if we had static checking for Suspense. I want TypeScript to force me to wrap my component in Suspense or tell the parent component to deal with it the same way Java makes me handle checked exceptions or force the caller to deal with it!